diktat-maven-plugin/src/main/kotlin/org/cqfn/diktat/plugin/maven/DiktatBaseMojo.kt

```diff
@@ -66,8 +66,8 @@ abstract class DiktatBaseMojo : AbstractMojo() {
      */
     override fun execute() {
         val configFile = resolveConfig()
```

Comment:
it's better to check for null in resolveConfig(). otherwise you are mixing the logic

Reply:
Fixed.